docs/DOCUMENTACION.md

```diff
@@ -14,15 +14,48 @@ A través de la web de la Junta de Andalucía podremos obtener los datos relativ
+- Deno: Mucho más seguro que Node.js ya que únicamente dota al programa a ejecutar los permisos que sean necesarios. No tiene un gestor de dependencias como Node tiene a npm ya que instala las dependencias a partir de una URL. Usa módulos ES6 en lugar de los CommonJs que usa Node. Usa promesas en vez de callbacks, lo que hace que el código sea más legible y reutilizable. Además dispone de de librerías o APIS del navegador, como fetch que no está disponible en Node. Deno supone todas estas ventajas con respecto a Node a cambio de un rendimiento un tanto inferior. Esta información ha sido obtenida del siguiente [enlace](https://openwebinars.net/blog/que-es-deno-y-que-lo-hace-diferente/#:~:text=Aunque%20Deno%20sigue%20la%20misma,el%20Event%20Loop%20de%20Deno.) o en su [página web](https://deno.land/).
+- Bun: Es un runtime relativamente nuevo que nos proporciona un rendimiento mucho mayor que Node y Deno. Se encuentra actualmente en fase Beta por lo que se pueden encontrar errores en algunas librerías de Node. Actúa además de runtime como gestor de dependencias y gestor de tareas. Podemos obtener más información respecto a Bun en el siguiente [enlace](https://dev.to/builderio/a-first-look-at-bun-is-it-really-3x-faster-than-nodejs-and-deno-45od) o en su [página web](https://bun.sh/).
+- Otras opciones menos conocidas son: Napa.JS , Hernes , JerryScript o QuickJS, pero no son muy usadas. 
+
+La lista de los runtimes descritos en este issue ha sido obtenida del siguiente [enlace](https://bestofjs.org/projects?tags=runtime). Como podemos observar, las tres primeras opciones nombradas tienen un uso mucho más alto que el resto, por eso no se ha hecho demasiado hincapié en ellas.
+En principio tenia pensado usar bun debido a su alta potencia y a que me parecía interesante ya que es un runtime "todo en uno", pero por errores de compatibilidad con mi versión de Sistema Operativo(cosas de estar en Beta) voy a usar NodeJS ya que lo he usado antes y quiero priorizar el rendimiento superior de NodeJs con respecto a las mejoras en seguridad que nos proporciona Deno.
+
+
+
+## Gestor de Dependencias
+Tras elegir el Runtime que se va a usar en la aplicación, es el turno de tomar la decisión de qué gestor de dependencias se va a usar. A continuación se muestran los más comunes:
+- NPM/PNPM: Gestor de dependencias de Node. Cuenta con un repositorio de más de 100000 paquetes y módulos.
+- Bower: Este gestor de paquetes(dependencias) corre sobre NPM. NPM es mucho más completo mientras que Bower apuesta por la reducción en el tamaño del archivo y los tiempos de carga para las dependencias del frontend. Se puede decir que está obsoleto ya que NPM puede hacer casi todo lo que hace Bower.
+- RequireJS: Es un cojunto de herramientas de JS que se puede utilizar para cargar módulos de manera sencilla incluyendo módulos de Node. Puede detectar automáticamente las dependencias necesarias en base a lo que estás utilizando.
```

Comment:
Por favor, no listéis cosas rándom y su descripción. Limitaros a establecer una serie de requisitos y poner una lista exhaustiva de los que cumplan esos requisitos.